### PR TITLE
German translation fastlane

### PR DESCRIPTION
**Pull request by Strubbl**

### CODE REVIEW

File: fastlane/metadata/android/de/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Eine selbst-gehostete Später-Lesen-App, die für dich zum komfortablen Lesen
+und Archivieren deiner Artikel erstellt wurde. Anders als andere Services,
+steht die Server-Software unter MIT-Lizenz, sodass du deinen eigenen Server
+aufsetzen kannst.: Befolge dazu die Anleitung unter
+[http://www.wallabag.org wallabag.org].
+
+Diese Android App erlaubt dir das Lesen und Verwalten deiner Aritkel und
+synchronisiert sie automatisch mit dem wallabag Server. Das nächste Mal,
+wenn du einen Artikel findest, den du später lesen möchtest, speichere ihn
+in wallabg. Er wird automatisch auf dein Telefon, Tablet, e-Reader oder
```
Comment (di72nn) [inline fastlane/metadata/android/de/full_description.txt:10]: "wallabg"

**Comment by di72nn** [comment]
I may be wrong, but I think F-Droid doesn't use changelogs unless they are tagged in the corresponding release.

**Comment by Strubbl** [comment]
> I may be wrong, but I think F-Droid doesn't use changelogs unless they are tagged in the corresponding release.

thanks, i did not know. if it's okay, we can merge the changelogs anyways

**Comment by di72nn** [comment]
Sure

**Pull request merged**